baselines/naming/src/v1beta1/naming_client.ts.baseline

```diff
@@ -1109,6 +1186,81 @@ export class NamingClient {
       callSettings
     ) as AsyncIterable<protos.google.protobuf.IEmpty>;
   }
+  // --------------------
+  // -- Path templates --
+  // --------------------
+
+  /**
+   * Return a fully-qualified projectExport resource name string.
+   *
+   * @param {string} project
+   * @param {string} exportParam
+   * @returns {string} Resource name string.
+   */
+  projectExportPath(project:string,exportParam:string) {
+    return this.pathTemplates.projectExportPathTemplate.render({
+      project: project,
+      exportParam: exportParam,
```

Comment:
I believe it should say `export` here, otherwise it won't get rendered properly.  You can do `"export"`, `gts` will remove the quotes if they are not needed.

Reply:
Cool. After couple trys with gax and even created a useless [PR](https://github.com/googleapis/gax-nodejs/pull/1202) in gax😢, finally, I got the idea. I only need to update the identifier in the method parameter. Update the PR and please take another look. Thanks for bearing with me. 🤝